Product/Production/Services/DocumentQueryCore/src/main/java/gov/hhs/fha/nhinc/docquery/deferred/impl/AdapterResponseHelper.java

```diff
@@ -36,4 +45,76 @@ public static AdhocQueryResponse createSuccessResponse() {
         return response;
     }
 
+    public static RegistryResponseType createRegistryResponseTypeWithXdsQuerySuccess() {
+        return createRegistryResponseTypeWithXdsQueryFailure();
```

Comment:
Should have dedicated code for a success message - not calling a Falure method with no args